system/system.proto

```diff
@@ -149,11 +152,24 @@ message RebootStatusRequest {
 }
 
 message RebootStatusResponse {
-  bool active = 1;      // If reboot is active.
-  uint64 wait = 2;      // Time left until reboot.
-  uint64 when = 3;      // Time to reboot in nanoseconds since the epoch.
-  string reason = 4;    // Reason for reboot.
-  uint32 count = 5;     // Number of reboots since active.
+  bool active = 1 [deprecated = true];  // If reboot is active.
+  uint64 wait = 2 [deprecated = true];  // Time left until reboot.
+  uint64 when = 3
+      [deprecated = true];  // Time to reboot in nanoseconds since the epoch.
+  string reason = 4 [deprecated = true];  // Reason for reboot.
+  uint32 count = 5 [deprecated = true];   // Number of reboots since active.
+
+  message Status {
+    types.Path subcomponent =
```

Comment:
Can we use component instead of sub-component?  A flat list of components should do the job.  Any use cases for hierarchy of components can be determined via gNMI.

Reply:
Component should be mandatory.  The component tree is very large on most devices.